keyboards/v60_type_r/keymaps/ifohancroft/config.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+// place overrides here
+#ifdef V60_POLESTAR
+#undef V60_POLESTAR
+#endif
+#undef USB_POLLING_INTERVAL_MS
```

Comment:
```suggestion
```

Reply:
I don't get it?
Don't I have to undefine USB_POLLING_INTERVAL_MS before defining it again?

Reply:
Done